fastlane/lib/fastlane/actions/crashlytics.rb

```diff
@@ -133,28 +133,27 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :notes,
                                        env_name: "CRASHLYTICS_NOTES",
                                        description: "The release notes as string - uses :notes_path under the hood",
-                                       optional: true,
-                                       is_string: true),
+                                       optional: true),
           FastlaneCore::ConfigItem.new(key: :groups,
                                        env_name: "CRASHLYTICS_GROUPS",
                                        description: "The groups used for distribution, separated by commas",
-                                       optional: true,
-                                       is_string: false),
+                                       skip_type_validation: true, # allows String, Array
```

Comment:
If I use `type: Array` then validation is failing for a single string...comma-separated string is okay

<img width="1114" alt="Screenshot 2021-06-11 at 19 22 30" src="https://user-images.githubusercontent.com/5364500/121726551-fcf4ed00-caea-11eb-8be8-31cbb20a200c.png">

Reply:
I _almost_ feel like this is a bug in `type: Array` 🤔 I see no issue with a single being valid 🤷‍♂️ Thoughts?